Core/include/Acts/TrackFitting/KalmanFitter.hpp

```diff
@@ -8,7 +8,6 @@
 
 #pragma once
 
-// Workaround for building on clang+libstdc++
```

Comment:
Why remove this? This include has been moved around frequently, causing issues. This comment indicates not to just that.

Reply:
Cosmetic change

Reply:
Fixed.